lib/embulk/input/google_analytics/plugin.rb

```diff
@@ -134,8 +134,7 @@ def run
           client = Client.new(task, preview?)
           columns = self.class.columns_from_task(task) + ["view_id"]
 
-          last_record_time = task['incremental'] && task["last_record_time"] ? Time.parse(task["last_record_time"]) : nil
-
+          last_record_time = Time.parse(task["last_record_time"]) if task['incremental'] && !task["last_record_time"].nil? && !task["last_record_time"].empty?
```

Comment:
As it included `activesupport`, `#blank?` is better than `#empty?`:
```suggestion
          last_record_time = Time.parse(task["last_record_time"]) if task['incremental'] && !task["last_record_time"].blank?
```

```irb
>> require "active_support/core_ext/time"
=> true
>> '   '.blank?
=> true
>> '   '.empty?
=> false
```

Reply:
Done